frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set-step/body.tsx

```diff
@@ -10,76 +10,130 @@ import {
   TextVariants,
   Tooltip,
 } from '@patternfly/react-core';
-import { ListPage } from '@console/internal/components/factory';
 import { Dropdown } from '@console/internal/components/utils';
-import { NodeModel } from '@console/internal/models';
 import { NodeKind } from '@console/internal/module/k8s';
-import { getName, MultiSelectDropdown } from '@console/shared';
+import { MultiSelectDropdown } from '@console/shared';
 import {
   deviceTypeDropdownItems,
+  diskModeDropdownItems,
   diskSizeUnitOptions,
   diskTypeDropdownItems,
 } from '@console/local-storage-operator-plugin/src/constants';
-import { NodesTable } from '@console/local-storage-operator-plugin/src/components/tables/nodes-table';
-import { diskModeDropdownItems, NO_PROVISIONER } from '../../../../constants';
 import { LocalVolumeSet, WizardDispatch, WizardState } from '../../reducer';
-
 import './body.scss';
+import { createWizardNodeState } from '../../../../utils/create-storage-system';
+import { SelectNodesTable } from '../select-nodes-table';
+import { NO_PROVISIONER } from '../../../../constants/common';
+
+export enum FilterDiskBy {
+  ALL_NODES = 'all-nodes',
+  SELECTED_NODES = 'selected-nodes',
+}
+
+const AllNodesLabel: React.FC<{ count: number }> = ({ count }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {t('ceph-storage-plugin~Disks on all nodes')}
+      {' ('}
+      {t('ceph-storage-plugin~{{nodes, number}} node', {
+        nodes: count,
+        count,
+      })}
+      {')'}
+    </>
+  );
+};
 
 export const LocalVolumeSetBody: React.FC<LocalVolumeSetBodyProps> = ({
   dispatch,
   state,
   storageClassName,
-  taintsFilter,
-  diskModeOptions = diskModeDropdownItems,
-  allNodesHelpTxt,
-  lvsNameHelpTxt,
-  deviceTypeOptions = deviceTypeDropdownItems,
+  nodes,
+  allNodes,
 }) => {
   const { t } = useTranslation();
+  const [radio, setRadio] = React.useState(FilterDiskBy.ALL_NODES);
+  const [activeMinDiskSize, setMinActiveState] = React.useState(false);
+  const [activeMaxDiskSize, setMaxActiveState] = React.useState(false);
+
+  React.useEffect(() => {
+    dispatch({ type: 'wizard/nodes', payload: allNodes });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
```suggestion
    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Please add a comment.

Reply:
On the first load of component , we should set the initial state.